### PR TITLE
ENH: coreg legend

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This PR adds a legend to the coreg app. It helps make the association fiducial <> color:

image

image

It's an item of #8833

### CODE REVIEW

**Comment by GuillaumeFavelier** [comment]
I pushed a commit to adjust the size so that it is visible with the `notebook` 3d backend too. Now it's 10% of the scene:

notebook | pyvistaqt
--|--
image | image

**Comment by GuillaumeFavelier** [comment]
This is ready for reviews from my end @agramfort, @larsoner

File: mne/gui/_coreg.py [hunk 1]
```diff
@@ -746,6 +747,11 @@ def _configure_picking(self):
             self._on_pick
         )
 
+    def _configure_legend(self):
+        mri_fids_legend_actor = self._renderer.legend(
+            labels=[("LPA", "red"), ("Nasion", "green"), ("RPA", "blue")])
```
Comment (agramfort) [inline mne/gui/_coreg.py:752]: can you avoid redefining here the colors of fiducials and define each color in only one place? besides can you use (0, 1, 0) for nasion in legend so it's more readable.

**Comment by GuillaumeFavelier** [comment]
@agramfort I reused the values in `DEFAULTS['coreg']`:

image

**Comment by GuillaumeFavelier** [comment]
I pushed a commit to fix:

```
E   ValueError: 
E   	Invalid color input: ([255.   0.   0.])
E   	Must be string, rgb list, or hex color string.  For example:
E   		color='white'
E   		color='w'
E   		color=[1, 1, 1]
E   		color='#FFFFFF'
```

**Review by agramfort — APPROVED** [review]

**Pull request merged**